src/client/clouds.cpp

```diff
@@ -191,25 +190,28 @@ void Clouds::updateMesh()
 
 		v2f p0 = v2f(xi,zi)*cloud_size + world_center_of_drawing_in_noise_f;
 
-		video::S3DVertex v[4] = {
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 0, 1),
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 1, 1),
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 1, 0),
-			video::S3DVertex(0,0,0, 0,0,0, c_top, 0, 0)
-		};
-
 		const f32 rx = cloud_size / 2.0f;
 		// if clouds are flat, the top layer should be at the given height
 		const f32 ry = is3D() ? m_params.thickness * BS : 0.0f;
 		const f32 rz = cloud_size / 2;
 
 		bool soft_clouds_enabled = g_settings->getBool("soft_clouds");
+
+		v3f pos(p0.X, m_params.height * BS, p0.Y);
+
+		video::S3DVertex v[4] = {
+		video::S3DVertex(0,0,0, 0,0,0, c_top, 0, 1),
+		video::S3DVertex(0,0,0, 0,0,0, c_top, 1, 1),
+		video::S3DVertex(0,0,0, 0,0,0, c_top, 1, 0),
+		video::S3DVertex(0,0,0, 0,0,0, c_top, 0, 0)
+		};
+
 		for (u32 i = 0; i < num_faces_to_draw; i++)
 		{
 			switch (i)
 			{
 			case 0:	// top
-				for (video::S3DVertex& vertex : v) {
```

Comment:
I would still keep the whitespace changes out of this.
(And thanks for bearing with us)